inc/components/class-image.php

```diff
@@ -257,7 +257,9 @@ public function configure( $picture ) {
 	 * @return string
 	 */
 	public function get_aspect_ratio( $size_config ) {
-		$aspect_ratio = $size_config['aspect_ratio'] ?? $this->config['aspect_ratio'] ?? false;
+		$aspect_ratio = isset( $size_config['aspect_ratio'] ) ?
```

Comment:
This should work... if you wanted to shorten, not sure it's 'simplification' ;)

```suggestion
		$aspect_ratio = ( $size_config['aspect_ratio'] ?? $this->config['aspect_ratio'] ) ?? false;
```